### DISCUSSION TITLE
Force spdlog file rotation

### DISCUSSION THREAD

**Discussion by htartisan** [gabime/spdlog#3024] [General]
If I setup spdlog with a "rotating file sink", is there a way for the calling application to "trigger/force" the file rotation operation?

**Comment by tt4g** [comment]
Duplicate #2997

rotating_file_sink cannot be operated from the application.
But you can create a custom sink with the features you desire: https://github.com/gabime/spdlog/wiki/4.-Sinks#implementing-your-own-sink
